packages/nc-gui/components/project/View.vue

```diff
@@ -1,9 +1,12 @@
 <script lang="ts" setup>
 import { useTitle } from '@vueuse/core'
 import NcLayout from '~icons/nc-icons/layout'
-const { openedProject } = storeToRefs(useBases())
+const basesStore = useBases()
+const { getProjectUsers } = basesStore
+const { openedProject, activeProjectId, baseUserCount } = storeToRefs(basesStore)
 const { activeTables } = storeToRefs(useTablesStore())
 const { activeWorkspace, workspaceUserCount } = storeToRefs(useWorkspace())
+import { isEeUI } from '#imports'
```

Comment:
put all import together. i.e. move this line after L3.

Reply:
Done